sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -83,6 +67,24 @@ Other noteworthy changes:
 - For a checkpoint store implementation using Azure Storage Blobs, use the
   [@azure/eventhubs-checkpointstore-blob](https://www.npmjs.com/package/@azure/eventhubs-checkpointstore-blob) package.
 
+Important note on checkpoints: `EventHubConsumerClient` does not support legacy checkpoint data.
+i.e. the checkpoints made using the package `@azure/event-processor-host`.
+In order to support interopability between different programming languages, a unifying checkpoint format was needed.
+This and the need to support improvements to the algorithm used for managing partition ownership made breaking changes necessary.
+
+### Sending events
+
+| In v2                              | Equivalent in v5                               | Sample                                                                                                                           |
+| ---------------------------------- | ---------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `EventHubClient.sendBatch(events)` | `EventHubProducerClient.sendBatch(eventBatch)` | [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/typescript/src/sendEvents.ts) |
+
+Other noteworthy changes:
+
+- The `send` method on the client is deprecated in favor of the `sendBatch` to encourage sending
+  events in batches for better throughput.
+- The `sendBatch` method on the client takes a object of type `EventDataBatch` that should be created
```

Comment:
Isnt there an overload that takes an array of events as well?

Reply:
I now mention this as well.